iml-manager-cli/src/nginx.rs

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) 2020 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+use crate::error::ImlManagerCliError;
+use console::Term;
+use lazy_static::*;
+use regex::{Captures, Regex};
+use std::{collections::HashMap, env, str};
+use structopt::StructOpt;
+use tokio::fs;
+
+#[derive(Debug, StructOpt)]
+pub enum NginxCommand {
+    /// Generate Nginx conf
+    #[structopt(name = "generate-config")]
+    GenerateConfig {
+        /// Set the nginx config path
+        #[structopt(short = "p", long = "path")]
```

Comment:
Nit: I guess this is a template path, so it could be `-i,--input`. And the help string could be `/// Template file`